cmd/catalog/main.go

```diff
@@ -70,7 +70,8 @@ var (
 	profiling = flag.Bool(
 		"profiling", false, "serve profiling data (on port 8080)")
 
-	installPlanTimeout = flag.Duration("install-plan-retry-timeout", 1*time.Minute, "time since first attempt at which plan execution errors are considered fatal")
+	installPlanTimeout  = flag.Duration("install-plan-retry-timeout", 1*time.Minute, "time since first attempt at which plan execution errors are considered fatal")
+	bundleUnpackTimeout = flag.Duration("bundle-unpack-timeout", catalog.DefaultBundleUnpackTimeout, "The time duration after which the bundle unpack Job for an installplan will be aborted and the installplan will be Failed. 0 is considered as having no timeout.")
```

Comment:
so this `bundleUnpackTimeout` is very similar to the `installPlanTimeout`-- I commented in @benluddy's timeout PR that we should have an exported constant for the default value of the timeout, but he disagreed saying there's no real reason to have that constant in the codebase if it's just used as a default for the flag and that's it. 

Based on that logic, WDYT about also just hardcoding the timeout? I don't see anywhere else in the PR where we use that DefaultBundleUnpackTimeout constant.

Reply:
Removed the constant.